.github/workflows/ci-release.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       contents: write
       deployments: write
     if: github.repository == 'jaegertracing/jaeger'
-    runs-on: ubuntu-latest
+    runs-on: equinix-4cpu-16gb
```

Comment:
I don't know if this has a bigger disk or not. Also, we need to be able to test the runner change before the release, which we cannot do with this workflow.

Reply:
which runner i should use ?

Reply:
I don't know yet, I am following up with CNCF admins on that. I am going to close this for now (this was not a help-wanted ticket btw, exactly for this reason).